src/services/services.ts

```diff
@@ -1725,7 +1725,7 @@ namespace ts {
 
         function getDocumentHighlights(fileName: string, position: number, filesToSearch: readonly string[]): DocumentHighlights[] | undefined {
             const normalizedFileName = normalizePath(fileName);
-            Debug.assert(filesToSearch.some(f => normalizePath(f) === normalizedFileName));
+            Debug.assert(filesToSearch.some(f => normalizePath(f) === normalizedFileName), `Could not find ${normalizedFileName} in ${filesToSearch}`);

```

Comment:
Uh, could this result in leaking filenames into telemetry? I might need to remove this.

Reply:
@amcasey

Reply:
Paths are generally okay, though we've had problems with things like `filesToSearch` being too long and causing volume problems.

Reply:
Wait really? I thought paths were a hard no. Many, if not _most_, will have usernames in them.

Reply:
I'll remove it, although I noticed that @amcasey added a `throw` of another path in getValidSourceFile a couple of years ago.